ambry-messageformat/src/main/java/com/github/ambry/messageformat/MessageFormatRecord.java

```diff
@@ -1826,6 +1836,42 @@ public static BlobData deserializeBlobRecord(CrcInputStream crcStream) throws IO
     }
   }
 
+  /**
+   * Get a summary of bytes in the given {@link ByteBuf}.
+   * @param bb THe {@link ByteBuf}.
+   * @return A summary of bytes in the byte buf.
+   */
+  static String byteBufSummary(ByteBuf bb) {
+    StringBuilder sb = new StringBuilder();
+    if (bb.readableBytes() <= 30) {
```

Comment:
There will unlikely be chunks with < = 30 bytes. Should we increase this limit? We might need full data to find out what's going wrong.

Reply:
Yes, we are now going to print the whole thing. We hit the crc error again yesterday but the logs were insufficient to conclude.